app/views/partials/channels-tab-request-matching.html

```diff
@@ -0,0 +1,106 @@
+<div class="form-group-2columns" style="margin-bottom: 15px;">
+  <div class="form-group" style="float: left; width: 75%; margin-bottom: 0px;" ng-class="{ 'has-error' : ngError.urlPattern }" ng-if="channel.type === 'http' || channel.type === 'polling'">
+    
+    <div class="form-group" style="float: left; width: 75%; margin-bottom: 0px;">
+      <label><span>*</span> URL Pattern</label>
+      <div class="input-group" style="width: 100%;">
+        <div ng-show="urlPattern.regex" class="input-group-addon">^</div>
+        <input class="form-control" placeholder="/example/path" type="text" ng-model="channel.urlPattern" ng-style="{ false: {'border-radius': '4px'}, true: {}}[urlPattern.regex];" />
+        <div ng-show="urlPattern.regex" class="input-group-addon">$</div>
+      </div>
+    </div>
+    <div class="addUrlRegexDelimiterDiv">
+      <label>
+        <input type="checkbox" ng-model="urlPattern.regex" style="margin-top: 0px; vertical-align: middle;" /> Auto-add<br />regex delimiters
+        <i ng-show="uiSettings.showTooltips" tooltip="This automatically adds start and ending anchors to the URL pattern regex. This is generally what you want." class="glyphicon glyphicon-question-sign tooltip-wide" style="font-size: 12px"></i>
+      </label>
+    </div>
+    <div class="has-error-msg" style="clear: both; margin-top: 5px;"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</div>
+
+  </div>
+  <div class="form-group" style="float: right; width: 23%; margin-bottom: 0px;" ng-class="{fullWidthPriority:channel.type === 'tcp' || channel.type === 'tls'}">
+    <label>
+      Priority
```

Comment:
The priority doesn't look too great when the url pattern is hidden
<img width="835" alt="screen shot 2016-03-17 at 09 26 38" src="https://cloud.githubusercontent.com/assets/1872071/13838122/816b4604-ec22-11e5-99b7-115b6cca89f5.png">

Is it possible to prevent it from filling the entire width?